src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -435,10 +453,32 @@
   {@context.upperCamelToLowerUnderscore(method.getSimpleName())}
 @end
 
-@private makeApiCall(method, request, isReturnEmptyMessage)
-  @@{@getMethodName(method)}.call({@request}, options)
-  @if isReturnEmptyMessage
-    nil
+@private makeApiCall(methodConfig, service)
+  @let request = requestParameterName(methodConfig)
```

Comment:
You may add 'method = methodConfig.getMethod' here and just use 'method' in this function.